GLMakie/assets/shader/distance_shape.frag

```diff
@@ -138,7 +138,7 @@ void main(){
     else if(shape == ROUNDED_RECTANGLE)
         signed_distance = rounded_rectangle(f_uv, vec2(0.2), vec2(0.8));
     else if(shape == RECTANGLE)
-        signed_distance = 1.0; // rectangle(f_uv);
```

Comment:
Hm, I remember I had all kind of artifacts with `rectangle(f_uv)` so I gave up and just used 1.0...
Will need to look at the reference images and try it out to see how bad it actually is.